### ISSUE TITLE
All pages are missing a `main` section

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#779]
## What
At the recent hack day, we added HTML to mark up the header and footer landmark areas of each page. The last major area we need to add is the "main" one. The main area must contain the main content of the page, i.e., not the header, footer, navigation bar, or any sidebars. On the [demo card page](https://civicplatform.org/cards/demo-card), this is the `<div class="css-o92fdl">` element.

Adding this element will provide the correct semantics to assistive technology and allow screen reader users to quickly move to the content. We can achieve a main element in one of two ways:

1. Using the HTML `<main>` element.
2. Adding the `role="main" attribute to the existing `<div>`.

The first option is better as it uses native HTML and meets the [First Rule Of ARIA Use](https://www.w3.org/TR/using-aria/#firstrule), which is to use HTML if there is a suitable element available.

While adding this, can we also add `tabindex="-1" and `id="main"` attributes to the element, and then the necessary code to hide the focus ring. We'll need these when we add a "skip link" (issue for this is forthcoming).

### HTML required
Either: `<main id="main" tabindex="-1">` or `<div class="css-o92fdl" id="main" role="main" tabindex="-1">`

### CSS required
`main:focus{outline:none}` (or different target element if using the `<div>`)

## When
It would be good to have this as soon as possible. If possible.

**Comment by jaronheard** [comment]
Add in PageLayout component
